packages/venia-concept/venia-upward.yml

```diff
@@ -109,6 +127,17 @@ resource:
     inline:
       template: './templates/notfound-shell.mst'
 
+assetManifest:
+  when:
+    - matches: env.NODE_ENV
+      pattern: 'development'
+      use: ./asset-manifest.json
```

Comment:
I'm seeing an error in the log with this configuration in dev mode: `Unrecognized configuration. Could not match a resolver to bundles: \'assetManifest.bundles\'`. Did you intend to check-in a hard-coded asset manifest, or should this file be generated and exist in the root?

I think the conditional fall through the bundles is masking this, or I'm misunderstanding the logging output.

Reply:
Disregard, there is no error in the log, I had a rogue `console.log` from some old debugging. I still don't see this file in my project, but it doesn't seem to be causing any problems in the app.

Reply:
Discussed with James, this is my fault. WebPack serves `./asset-manifest.json` via virtual filesystem, that is only started up with `yarn watch:` and not `yarn stage:` commands.